docs/src/main/asciidoc/resteasy-reactive.adoc

```diff
@@ -324,6 +324,18 @@ but they require you to specify the parameter name.
 
 See <<parameter-mapping>> for more advanced use-cases.
 
+[NOTE]
+====
+When an exception occurs in RESTEasy Reactive request parameter handling code, the exception is not printed by default to the log (for security reasons).
+This can sometimes make it hard to understand why certain HTTP status codes are returned (as the JAX-RS mandates the use of non-intuitive in various cases).
```

Comment:
```suggestion
This can sometimes make it hard to understand why certain HTTP status codes are returned (as the JAX-RS mandates the use of non-intuitive error codes in various cases).
```